src/name/resolve.js

```diff
@@ -62,28 +62,31 @@ module.exports = (createCommon, options) => {
     it('should not get the entry if its validity time expired', function (done) {
       this.timeout(50 * 1000)
 
-      const value = fixture.cid
       const publishOptions = {
         resolve: true,
-        lifetime: '1ms',
+        lifetime: '100ms',
         ttl: '10s',
-        key: 'self'
+        key: 'self',
+        'allow-offline': true
       }
 
-      ipfs.name.publish(value, publishOptions, (err, res) => {
+      ipfs.add(Buffer.from('should not get the entry if its validity time expired'), (err, r) => {
```

Comment:
Can you add a comment explaining why we do this new `ipfs.add`? 

As we intend to publish a record with less lifetime than the one published on the previous test, `go-ipfs` will return an error for this. I am also thinking if we should just do the `beforeEach` before.

Reply:
i just prefer to not re-use stuff between tests.

can you explain the second part further? go-ipfs passes this test.